cobra/flux_analysis/sampling.py

```diff
@@ -355,10 +359,11 @@ def _sample_chain(args):
 
     center and n_samples are updated locally and forgotten afterwards.
     """
-    sampler, n = args       # has to be this way to work in Python 2.7
+    sampler, n, idx = args       # has to be this way to work in Python 2.7
     center = sampler.center
+    np.random.seed(int(time() * idx + idx) % np.iinfo(np.int32).max)
```

Comment:
Honestly, it would probably would in most cases. The thing is that it may use system time and processes may be spawned at the same time. So if the clock resolution is low all processes might be seeded to the same value in the worst case. The code here just ensures that this is impossible even though I agree it's a bit convoluted...